app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.html

```diff
@@ -45,9 +45,12 @@
           </a>
         </td>
         <td>
-          <a class="heavy" ng-if="application.build.buildNumber" href="{{application.build.url}}" target="_blank">
+          <a class="heavy" ng-if="application.build.buildNumber && !application.build.images" href="{{application.build.url}}" target="_blank">
             <span>#</span>{{application.build.buildNumber}}
           </a>
+          <ul ng-if="application.build.images" style="list-style: none" ng-repeat="image in application.build.images">
```

Comment:
We use inline styling in a bunch of places, but `class="list-unstyled"` feels like the better option here.